src/service/ui/legacyMessaging.js

```diff
@@ -40,6 +40,7 @@ const Dialog = GObject.registerClass({
     _init(params) {
         super._init({
             application: Gio.Application.get_default(),
+            title: _('Send SMS'),
```

Comment:
This is already set in the template, setting it here is redundant and makes maintenance harder:

https://github.com/GSConnect/gnome-shell-extension-gsconnect/blob/3758807fac3f2b3c09a30a1767cf7f551d27d915/data/ui/legacy-messaging-dialog.ui#L15